target/scripts/build/packages.sh

```diff
@@ -82,12 +77,17 @@ function _install_packages() {
     opendmarc libsasl2-modules sasl2-bin
   )
 
+  DEBUG_PACKAGES=(
+    bind9-dnsutils ed iproute2 iputils-ping most
```

Comment:
I didn't know what `ed` is for. The internet told me, it's the oldest unix editor. I am wondering if it's just me, not knowing/using ed?

Here we could switch to something more modern, like `vim`.

Reply:
I actually looked into that a month or so ago, it was added a long time ago and isn't necessary.

`vim` isn't noob friendly IMO. If you want to include a text editor use `nano`?

Reply:
Noobs shouldn't edit files in a running container 😆

You might be right, I don't really care if `vim` or `nano` is included 👍

Reply:
This will be resolved with another PR to have a separation of concerns here.